### DISCUSSION TITLE
Airflow Scheduler berjalan tiap detik

### DISCUSSION THREAD

**Discussion by takdirzd** [teamalgoritma/community#914] [Q&A] [answered]
saya memiliki apache airflow yang terinstall di linux debian server. saya menginstall airflow standalone (tanpa docker). saya memiliki codingan untuk mengirim email. tapi ketika saya sudah membuat dagsnya dan ketika run airflow scheduler, malah schedulernya berjalan terus tanpa mengikuti scheduler yang sudah sy tetapkan di dalam .py/dagsnya, jadi email akan masuk terus menerus, yang dimana saya hanya setting tiap jam 8 setiap hari.

yang salah dimana ya? atau ada settingan tertentu yang harus dilakukan di airflow.cfg nya? krn wktu install, sy membuat sendiri file airflow.cfg nya

![DAGS](https://github.com/user-attachments/assets/23c7b851-3063-4e1d-99bc-02be87f86cff)
di atas adalah settingal scheduler di dagsnya,. (untuk waktunya hanya testing, pada intinya saya mau runningnya tiap jam tersebut dan setiap hari

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @takdirzd ,

Dari yang terlihat, tidak ada configurasi yang salah untuk CRON. Tapi ada beberapa kemungkinan hal ini terjadi:

- Mungkin DAG mencoba untuk mengejar start_date hingga waktu saat ini walaupun `catchup=False`. Anda dapat memindahkan `start_date` dari default_args ke dalam parameter DAG.
- Mungkin `airflow.cfg` terdapat pengaturan yang berubah, sehingga Anda sebaiknya untuk membuat proyek airflow baru di dalam mesin Anda dengan `airflow db migrate` kemudian menerapkan lagi DAG yang telah Anda buat.

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Comment by adizaen** [comment]
Dear @takdirzd 

Terima kasih telah bertanya melalui Github discussions Algoritma. Untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.

**Best regards,**
Team Algoritma
